### ISSUE TITLE
[BUG] Search and Explore malfunctioning with disabled Machine Learning on v1.91.0

### ISSUE THREAD

**Issue by pig-cop** [immich-app/immich#5782]
### The bug

When Machine Learning is disabled Smart Search will remain the default search mode and searching for metadata values without adding 'm:' will produce a 'Failed to search (500 - Internal Server Error)' error.

Additionally with the new version all the contents in the Explore page are vanished.
Even without Machine Learning before I was able to access Recents and the other categories from there, now there's only an empty page.
![Screenshot 2023-12-17 at 10-51-55 Explore](https://github.com/immich-app/immich/assets/1793444/e290ed08-613e-4c38-a3f7-1810a9b29208)

### The OS that Immich Server is running on

Debian

### Version of Immich Server

v1.91.2

### Version of Immich Mobile App

v1.91.0 build.115

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
version: "3.8"

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${TAG}
    command: [ "start.sh", "immich" ]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - stack.env
    ports:
      - 2283:3001
    depends_on:
      - redis
      - database
    restart: always

  immich-microservices:
    container_name: immich_microservices
    image: ghcr.io/immich-app/immich-server:${TAG}
    command: [ "start.sh", "microservices" ]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - stack.env
    depends_on:
      - redis
      - database
    restart: always

  redis:
    container_name: immich_redis
    image: redis:6.2
    restart: always

  database:
    container_name: immich_postgres
    image: tensorchord/pgvecto-rs:pg14-v0.1.11
    env_file:
      - stack.env
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      PG_DATA: /var/lib/postgresql/data
    volumes:
      - pgdata:/var/lib/postgresql/data
    restart: always

volumes:
  pgdata:
  model-cache:
```

### Your .env content

```Shell
IMMICH_MACHINE_LEARNING_ENABLED = false
```

### Reproduction steps

```bash
1. Disable Machine Learning my removing the container from docker-compose.yml
2. (Optional) Add IMMICH_MACHINE_LEARNING_ENABLED environment variable, set it to false
3. Check Search and Explore from Web
```

**Comment by alextran1502** [comment]
will be fixed in #5747

**Comment by pig-cop** [comment]
I know this is considered closed but a part of what I described is still bugged to this day in version 1.105.1.

When Machine Learning is disabled the Explore Page remains empty where it was at least displaying the "Recents" section before:

![image](https://github.com/immich-app/immich/assets/1793444/08cb12af-ce7b-4c67-9f85-efde0d8d9cab)
